src/main/java/uk/gov/mca/beacons/service/model/Registration.java

```diff
@@ -3,6 +3,11 @@
 import java.util.List;
 import javax.validation.Valid;
 
+/**
+ * Class representing a registration object.
+ *
+ * TODO: Determine how to persist reference number and registration object.
+ */
 public class Registration {
 
   @Valid
```

Comment:
Robyn and I talked about this last week.  I believe his view is that only the composite entities of the `Registration` are persisted, with the reference number placed on the `Beacon`.  The `Registration` itself just remains as an abstraction.